docs/build/micro-rdk/data_management/_index.md

```diff
@@ -0,0 +1,176 @@
+weight: 30
+type: "docs"
+description: "Configure data capture and sync to save data from components."
+images: ["/services/icons/data-capture.svg"]
+icon: true
+tags: ["data management", "cloud", "sync"]
+no_list: true
+no_service: true64
+# SMEs: Gautham V.
+---
+
+Micro-RDK's data management service captures data from one or more components in the ESP32's memory. The service periodically uploads data to Viam cloud. If the machine restarts all data that hasn't been synced will be lost.
+
+The service can capture data from multiple components at the same or different frequencies.
+Micro-RDK does not impose an upper limit on the frequency of data collection.
+However, in practice, high frequency (> 100Hz) requires special considerations on the ESP32.
+
+## Add the data management service
+
+1. Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+2. Click the **+** icon next to your machine part in the left-hand menu and select **Service**.
+3. Select the `data management` type, then either use the suggested name or specify a name for your data management service, for example `data-manager`.
+4. Click **Create**.
+5. On the panel that appears, you can manage the capturing and syncing functions and specify the sync **Interval**
+   {{< alert title="Info" color="info" >}}
+   With Micro-RDK the **Directory**, **Tags**, and **Additional paths** fields are ignored and should not be configured.
+   {{< /alert >}}
+6. Click the **Save** button in the top right corner of the page.
+
+![data capture configuration](/tutorials/data-management/data-management-conf.png)
+
+{{%expand "Click to view the JSON configuration for the data management service" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [],
+  "services": [
+    {
+      "attributes": {
+        "capture_dir": "",
+        "tags": [],
+        "additional_sync_paths": [],
+        "sync_interval_mins": 3
+      },
+      "name": "my-data-manager",
+      "namespace": "rdk",
+      "type": "data_manager"
+    }
+  ]
+}
+```
+
+{{% /expand%}}
+
+## Configure data capture for individual components
+
+Once you have added the data capture service, you can specify the data you want to capture at a component level.
+
+### Supported components
+
+Only the following components types are currently supported with data capture:
+|Type | Method |
+|----- | ----------- |
+|Sensor | GetReadings |
+|Movement Sensor | AngularVelocity, LinearAcceleration, LinearVelocity |
+
+To add data capture for a component, navigate to the **CONFIGURE** tab of your machine's page in the Viam app.
+
+For each component you can capture data for, there is a `Data capture` section in its panel.
+Click `Add Method` and then select the **Method** type and the capture **Frequency**.
+
+{{< alert title="Caution" color="caution" >}}
+
+Avoid configuring data capture to higher rates than your hardware can handle, as this leads to performance degradation.
```

Comment:
Nit: some clients might have quantum computers!

```suggestion
Avoid configuring data capture to higher rates than your hardware can handle, as this can lead to performance degradation.
```